oops/hosts/galileo/ssi/__init__.py

```diff
@@ -214,14 +214,18 @@ def __init__(self, meta_dict):
         self.mode = meta_dict['TELEMETRY_FORMAT_ID']
 
         # Window
+        self.window = None
         if 'CUT_OUT_WINDOW' in meta_dict:
-            self.window = np.array(meta_dict['CUT_OUT_WINDOW'])
-            self.window_origin = self.window[0:2]-1
-            self.window_shape = self.window[2:]
-            self.window_uv_origin = np.flip(self.window_origin)
-            self.window_uv_shape = np.flip(self.window_shape)
-        else:
-            self.window = None
+            window = np.array(meta_dict['CUT_OUT_WINDOW'])
+            
+            # check for [-1,-1,-1,-1].  This is the value written in the 
+            # supplemental index when there is no CUT_OUT_WINDOW in the label.
+            if np.all(window != -1):
```

Comment:
Is there ever an occasion where -1 appears in CUT_OUT_WINDOW without all four values being -1? Because this will not execute if there's a single occurrence of -1. Just making sure that's what you mean. If the only possible case is (-1,-1,-1,-1), then the most painfully explicit thing to write would be
```
if window != (-1,-1,-1,-1):
```
Also, what do you want to happen if the CUT_OUT_WINDOW really is (-1,-1,-1,-1)? Right now, the attributes are not filled in. Is (-1,-1,-1,-1) really equivalent to (1, 800, 1, 800)?

Reply:
I'm pretty sure this never happens the GLL SSI data images (but I'm checking).  Either all the elements are >=0 or they are all -1.  In the case of (-1,-1,-1,-1), self.window is left as None, and the image does not get cropped.

Reply:
I have confirmed that there are no cases in GLL SSI where CUT_OUT_WINDOW contains any -1 values unless all values are -1.  Changed to  

> if window.tolist() != [-1,-1,-1,-1]

to exactly correspond to the comment.